src/screens/SSIPersonalDataScreen/index.tsx

```diff
@@ -117,14 +117,16 @@ class SSIPersonalDataScreen extends PureComponent<IProps, IState> {
                   placeholderValue={translate('last_name_placeholder')}
                 />
               </TextInputContainer>
-              <SSITextInputField
-                label={translate('email_address_label')}
-                maxLength={EMAIL_ADDRESS_MAX_LENGTH}
-                autoComplete={'email'}
-                onChangeText={this.onEmailAddressChange}
-                onEndEditing={this.onEmailAddressValidation}
-                placeholderValue={translate('email_address_placeholder')}
-              />
+              <TextInputContainer>
```

Comment:
this needs to be removed as you now added a bottom margin

Reply:
I wondered why there's no `TextInputContainer` there :)
removed